web-components/src/components/tabs/Tabs.ts

```diff
@@ -844,6 +844,7 @@ export namespace Tabs {
           break;
         }
         case Key.Enter:
+        case Key.NumpadEnter:
         case Key.Space: {
```

Comment:
numpad enter should also active. These should probably be added to a utility function like isActionKeypress